data/libraries.json

```diff
@@ -803,5 +803,14 @@
             "JavaScript"
         ],
         "url": "http://ivan.vucica.net"
+    },
+     {
+        "doap": null,
+        "last_renewed": null,
+        "name": "Stanza",
+        "platforms": [
+            "TypeScript"
+        ],
+        "url": "http://stanza,io"
```

Comment:
I think you should replace the , with a .

Reply:
Resolved, Seems I'lI need another glasses :)